src/interpreter/ByteCode.cpp

```diff
@@ -80,6 +80,7 @@ void SetGlobalVariable::dump(const char* byteCodeStart)
 ByteCodeBlock::ByteCodeBlock(InterpretedCodeBlock* codeBlock)
     : m_isEvalMode(false)
     , m_isOnGlobal(false)
+    , m_shouldClearStack(false)
```

Comment:
This member variable is defined but not used. Is it going to be used later?

Reply:
The member variable is used on FunctionObjectInlines.h `if (UNLIKELY(blk->m_shouldClearStack)) {`

Reply:
Oh, I missed that. Thanks.